opennms-config-jaxb/src/main/java/org/opennms/netmgt/config/poller/PollerConfiguration.java

```diff
@@ -298,7 +323,9 @@ public boolean equals(Object o) {
                 && Objects.equals(getDefaultCriticalPathRetries(), that.getDefaultCriticalPathRetries())
                 && Objects.equals(getNodeOutage(), that.getNodeOutage())
                 && Objects.equals(getPackages(), that.getPackages())
-                && Objects.equals(getMonitors(), that.getMonitors());
+                && Objects.equals(getMonitors(), that.getMonitors())
+                && Objects.equals(getAsyncPollingEngineEnabled(), that.getAsyncPollingEngineEnabled())
+                && Objects.equals(getMaxConcurrentAsyncPolls(), that.getMaxConcurrentAsyncPolls());
```

Comment:
Here's a fun question -- are config objects "equal" if one explicitly sets max concurrent to `100` and the other is null but gets it implicitly?

Arguably they are _not_ equal, because you're serializing the fields, and they wouldn't write out equivalent XML or whatever.

Reply:
Agreed. This behaves the same way as other properties in the object do currently though, hesitant to change that in this scope.

Reply:
I dunno, it's develop, seems like that's the time to do it...